chromadb/utils/data_loaders.py

```diff
@@ -0,0 +1,23 @@
+import importlib
+from typing import Optional, Sequence, List
+import numpy as np
+from chromadb.api.types import URI, DataLoader, Image
+from concurrent.futures import ThreadPoolExecutor
+
+
+class ImageLoader(DataLoader[List[Optional[Image]]]):
+    def __init__(self, max_workers: int = 4) -> None:
```

Comment:
Can we set max_workers to be multiprocessing.cpu_count() by default

https://github.com/chroma-core/chroma/blob/b26e0e848886c063a79a3e57067114452cc5ea73/chromadb/segment/impl/vector/hnsw_params.py#L61